recipes/r-rmatio/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = '0.16.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmatio
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rmatio_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rmatio/rmatio_{{ version }}.tar.gz
+  sha256: 73d2d6c4db1cf25535ce3e87a199f700dca338ed0e5d4cfc6484f3d2cd3828ab
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*'
```

Comment:
```suggestion
```

Reply:
Thanks so much for the review @dbast!

Before I set missing_dso_whitelist, I encountered [this error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=524241&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=9410) in the build for OSX and Linux. 

I followed the advice [here](https://gitter.im/conda-forge/conda-forge.github.io?at=5deeaa8355066245989a31be) that led to [this PR](https://github.com/conda-forge/staged-recipes/pull/10407) where `missing_dso_whitelist` is set. Do you think I the builds would fail if we remove these two lines? I appreciate any advice!

Reply:
the whitelisting can be removed with libzlib in host requirements. whitelisting * hides potential linking issues, so please remove.